WalletWasabi/Blockchain/TransactionBroadcasting/TransactionBroadcaster.cs

```diff
@@ -109,22 +120,27 @@ public async Task<BroadcastingResult> BroadcastAsync(SmartTransaction tx)
 		} while (completedTask.IsFaulted && tasksToWaitFor.Count > 0);
 
 		var results = await Task.WhenAll(broadcastToNodeTasks).ConfigureAwait(false);
-		return results.SequenceResults().ThenError<BroadcastError>(e => new BroadcastError.AggregatedErrors(e));
+		var errors = results
+			.Select(r => r.Match(_ => (IsError: false, Error: null), e => (IsError: true, Error: e)))
```

Comment:
There is a null warning here

Reply:
it doesn't matter because it is never used.

Reply:
ok, I agree that it would be better to not having it.